Testshell/test_shell.cpp

```diff
@@ -95,4 +94,49 @@ class TestShell {
     return Command{userInput,
                    vector<string>()}; // Simplified parsing for demonstration
   }
+
+  void help() {
+    printHeader();
+    printTeamInfo();
+    printCommands();
```

Comment:
사용가능한 Test Script가 무엇이 있는 지 출력해주는 함수도 추가하면 좋을 거 같습니다.
```suggestion
  printCommands();
  printTestScripts();
```

Reply:
누락된 부분 알려주셔서 감사합니다!
추가 완료하였습니다.